deps/wazuh_testing/wazuh_testing/modules/logcollector/event_monitor.py

```diff
@@ -0,0 +1,165 @@
+import re
+
+from wazuh_testing import T_30, T_60
+from wazuh_testing.modules.logcollector import LOG_COLLECTOR_PREFIX, MACOS_LOG_COMMAND_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools import LOG_FILE_PATH
+
+
+def make_logcollector_callback(pattern, prefix=LOG_COLLECTOR_PREFIX, escape=False):
+    """Create a callback function from a text pattern.
+
+    It already contains the analsisd prefix.
+
+    Args:
+        pattern (str): String to match on the log.
+        prefix (str): regular expression used as a prefix before the pattern.
+        escape (bool): Flag to escape special characters in the pattern
+
+    Returns:
+        lambda: function that returns if there's a match in the file
+
+    Examples:
+        >>> callback_bionic_update_started = make_vuln_callback("Starting Ubuntu Bionic database update")
```

Comment:
Update example to use logcollector

Reply:
Done in [here](https://github.com/wazuh/wazuh-qa/pull/3486/commits/40e727636c5f3938a6430afe75851cecd436d81c)